service/src/main/java/com/derpgroup/derpwizard/resource/HomeResource.java

```diff
@@ -37,7 +37,7 @@
  * @since 0.0.1
  */
 @Path("/")
-@Produces(MediaType.APPLICATION_JSON)
+@Produces(MediaType.TEXT_PLAIN)
```

Comment:
Sure.  This was just faster, and I figured that it's theoretically feasible that we'd have endpoints that return something other than JSON.